src/lib_components/components/NeonContext/NeonContext.jsx

```diff
@@ -259,7 +293,7 @@ const Provider = (props) => {
      Render
   */
   return (
-    <Context.Provider value={[state]}>
+    <Context.Provider value={[state, dispatch]}>
```

Comment:
Just something to be aware of here... part of the design of NeonContext is that you can *only* get state.

This was a simplified way of preventing other components from having the ability to mess with what's in the context. The perceived risk is that the NeonContext is often shared across arbitrarily many components, often existing at the page level, and it seemed leaky to allow any consumer of the context at any level be able to dispatch arbitrary changes to it.

I could see this going a few ways:

1. This safeguard isn't that important (it only becomes a problem if we make it a problem, after all)
1. This safegaurd is important so we preserve it by one of:
    1. Update any consumer of this provider currently reliant on dispatch to not require dispatch... somehow?
    1. On this line don't provide the full-powered dispatch but an exportable wrapped dispatch that only allows certain actions